apps/meteor/app/slashcommands-invite/server/server.ts

```diff
@@ -60,10 +60,15 @@ slashCommands.add({
 		await Promise.all(
 			usersFiltered.map(async (user) => {
 				try {
-					return await addUsersToRoomMethod(userId, {
-						rid: message.rid,
-						users: [user.username || ''],
-					});
+					const inviter = await Users.findOneById(userId);
+					return await addUsersToRoomMethod(
+						userId,
+						{
+							rid: message.rid,
+							users: [user.username || ''],
+						},
+						inviter ?? undefined,
```

Comment:
inviter ?? undefined  is the same thing than inviter.

since this is an slash command, even with the method allowing undefined inviters I think you should thrown an error if the inviter is undefined